<launch>

  <node pkg="asl_turtlebot" type="supervisor.py" name="turtlebot_supervisor" output="screen"/>
  <node pkg="asl_turtlebot" type="pose_controller.py" name="turtlebot_pose_controller" />
  <node pkg="asl_turtlebot" type="navigator.py" name="turtlebot_navigator" />
  <!--<node pkg="asl_turtlebot" type="detector.py" name="turtlebot_detector" />-->
</launch>
